gammapy/irf/effective_area.py

```diff
@@ -252,7 +252,7 @@ def from_parametrization(cls, energy_axis_true=None, instrument="HESS"):
         pars = {
             "HESS": [6.85e9, 0.0891, 5e5],
             "HESS2": [2.05e9, 0.0891, 1e5],
-            "CTA": [1.71e11, 0.0891, 1e5],
+            "CTAO": [1.71e11, 0.0891, 1e5],
```

Comment:
This is a backward incompatible change. Add a deprecation warning in case `"CTA"` is called.

Reply:
this is not a deprecated class, not a rename of an argument, not a deprecated attribute. This is the deprecation of a parameter of accepted values of an argument...
I have then inserted a piece of code